CHANGELOG.md

```diff
@@ -8,33 +8,55 @@ This projects adheres to [Semantic Versioning](https://semver.org/) and [Keep a
+- `WPThemeReview.PluginTerritory.ForbiddenFunctions`: checks if a theme is using functions that fall under a plugin territory. [Handbook rule](https://make.wordpress.org/themes/handbook/review/required/#presentation-vs-functionality).
+- `WPThemeReview.PluginTerritory.NoAddAdminPages`: checks if a theme is using `add_..._page()` functions, with the exception of `add_theme_page()`. [Handbook rule](https://make.wordpress.org/themes/handbook/review/required/theme-check-plugin/#admin-menu).
+- `WPThemeReview.PluginTerritory.SessionFunctionsUsage`: taken from `WordPress-VIP` ruleset, which will be deprecated in the `WPCS` version 2.0. Prevents the usage of the session functions in themes.
+- `WPThemeReview.PluginTerritory.SessionVariableUsage`: taken from `WordPress-VIP` ruleset, which will be deprecated in the `WPCS` version 2.0. Prevents the usage of the session variables in themes.
+- `WPThemeReview.Plugins.CorrectTGMPAVersion` Verifies that if the [TGM Plugin Activation](http://tgmpluginactivation.com/) library is included, the correct version is used. [Handbook rule](https://make.wordpress.org/themes/handbook/review/required/theme-check-plugin/#admin-menu).
+- `WPThemeReview.ThouShallNotUse.ForbiddenIframe`: checks if a theme is using `<iframe>`. [Handbook rule](https://make.wordpress.org/themes/handbook/review/required/theme-check-plugin/#info).
+- `WPThemeReview.ThouShallNotUse.NoAutoGenerate`: checks if a theme has been generated using theme generators.
+
+#### Other sniffs in the ruleset
+- `PHPCompatibilityWP`: added as an entire ruleset, checking that the theme is compatible with PHP 5.2 and above
+- `Generic.PHP.DisallowShortOpenTag`: prohibits the usage of the PHP short open tags. [Handbook rule](https://make.wordpress.org/themes/handbook/review/required/theme-check-plugin/#additional-checks).
+- `Generic.PHP.DisallowAlternativePHPTags`: disallows the usage of alternative PHP open tags (`<%` and similar).
+- `Squiz.WhiteSpace.SuperfluousWhitespace.StartFile`: files starting with a PHP open tag shouldn't have a whitespace preceding it, to prevent possible `headers already sent` errors.
+- `PSR2.Files.ClosingTag.NotAllowed`: files should omit the closing PHP tag at the end of a file, to prevent possible `headers already sent` errors.
+- `Internal.LineEndings.Mixed`: mixed line endings are not allowed. [Handbook rule](https://make.wordpress.org/themes/handbook/review/required/theme-check-plugin/#line-endings).
+- `Internal.Tokenizer.Exception`: minified scripts or files should have original files included. [Handbook rule](https://make.wordpress.org/themes/handbook/review/required/#stylesheets-and-scripts).
+- `Generic.Files.ByteOrderMark`: no ByteOrderMark allowed - important to prevent issues with content being sent before headers
+- `Generic.CodeAnalysis.EmptyStatement`: PHP tags without anything between them is just sloppy
```

Comment:
Missing full stop `.` at the end of the sentence.

And, while I recognize my own words here, this may not be the best phrasing for the changelog :wink:

Reply:
I agree. I can change this to: 

> prohibits empty statements in the code (PHP tags with no content or only comments in them).

sounds good?

Reply:
Hang on - There are two sniffs related to this - `Generic.CodeAnalsysis.EmptyStatement` and `Generic.CodeAnalysis.EmptyPHPStatement` (PR https://github.com/squizlabs/PHP_CodeSniffer/pull/1814 and already in WPCS as `WordPress.CodeAnalysis.EmptyStatement`) - and I just realized that:
1. WPTRTCS only includes one of them.
2. The ruleset inline documentation is wrong in this case as it refers to the other sniff.

So I think we should decide if we want to add the second sniffs as well.
* `Generic.CodeAnalsysis.EmptyStatement` (currently included) checks for empty control structure leafs, i.e.
    ```php
    if ($a = $b) {
        // This would be reported.
    } else {
        $b = $a;
    }
    ```
* `Generic.CodeAnalysis.EmptyPHPStatement`/`WordPress.CodeAnalysis.EmptyStatement` checks for empty PHP tags and double semi-colons, like `statement; ;`.

Reply:
I'm definitely for including the `Generic.CodeAnalsysis.EmptyStatement` and `WordPress.CodeAnalysis.EmptyStatement`. 
Once those are added I can include those in the changelog.